Form/Type/ModelType.php

```diff
@@ -51,7 +51,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         if ($options['multiple']) {
             if (array_key_exists('choice_loader', $options) && $options['choice_loader'] !== null) { // SF2.7+
-                $builder->addViewTransformer(new ModelsToArrayTransformer($options['choice_list'], $options['model_manager'], $options['class']), true);
+                $builder->addViewTransformer(new ModelsToArrayTransformer(
+                    $options['choice_loader'], 
+                    $options['model_manager'], 
```

Comment:
There are trailing spaces at then end of lines 54 and 55. I think you can configure phpstorm to remove them on save.